marathon-bigip-ctlr.py

```diff
@@ -361,7 +361,7 @@ def healthcheck_sendstring(data):
     Args:
         data: Health Monitor dict
     """
-    if data['protocol'] == "http":
+    if 'type' in data and data['type'] == "http":
```

Comment:
Just curious, in the send_string below, do we need to urlencode the path?

Reply:
I will investigate and create an issue if necessary.